<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QWidget" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>صفحه ثبت نام</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ثبت نام</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="formWidget" native="true">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <property name="rowWrapPolicy">
       <enum>QFormLayout::DontWrapRows</enum>
      </property>
      <property name="labelAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="formAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="horizontalSpacing">
       <number>7</number>
      </property>
      <property name="verticalSpacing">
       <number>15</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>نام</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>نام کاربری</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>رمز عبور</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>تکرار رمز عبور</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>ایمیل</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="nameLE"/>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="usernameLE">
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="passLE">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="repassLE">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLineEdit" name="emailLE"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="horizontalWidget" native="true">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <property name="leftMargin">
       <number>180</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="back_pbn">
        <property name="text">
         <string>بازگشت</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="register_pbn">
        <property name="text">
         <string>ثبت اطلاعات</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
